net_util/src/queue_pair.rs

```diff
@@ -288,6 +292,44 @@ impl RxVirtio {
     }
 }
 
+#[derive(Default, Clone)]
+struct IovecBuffer(Vec<libc::iovec>);
+
+unsafe impl Send for IovecBuffer {}
```

Comment:
Send and Sync are usually auto implemented, but not for pointer(libc::iovec contains pointer).
As an empty container, it can be safely move ownership and safely shared between threads.

Reply:
I know, but without a comment your code won't pass the CI check. We require safety comments for all unsafes.